CMakeLists.txt

```diff
@@ -1,6 +1,18 @@
 #---Check if cmake has the required version-----------------------------------------------------
+
 cmake_minimum_required(VERSION 3.4.3 FATAL_ERROR)
-cmake_policy(SET CMP0005 NEW)
+
+#---CMake policy settings-----------------------------------------------------------------------
+# See "cmake --help-policy <POLICY>" for more details
+
+cmake_policy(VERSION 3.4.3)
+
+foreach(policy CMP0068)
```

Comment:
Can we make a list out of this to make it more explicit that this intended to be a is a list of new policies? Like:
```CMake
# Add policies which should use the NEW behavior if available here:
set(new_policies CMP0068)
foreach(policy ${new_policies})
  if(POLICY ${policy})
    cmake_policy(SET ${policy} NEW)
  endif()
endforeach()
```